_docs/_latest/develop/dev-setup.md

```diff
@@ -86,19 +50,17 @@ Install openssl:
   $ brew install openssl
   ```
 
-Install mysql_config
+Install mysql_config:
 
   ```bash
   $ brew install mysql
   ```
 
-### Ubuntu setup
-
-Ubuntu users can reference and [install the apt packages here](https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-dev/install/dependencies/apt_packages.txt).
-
 ### Creating a virtualenv
 
-Ensure virtualenv is installed in your system.
+Ensure virtualenv is installed in your system.  Virtualenv allows you to
```

Comment:
nit: One space after a period, not two. Please fix everywhere.

Reply:
done